validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorApiChannel.java

```diff
@@ -60,36 +60,42 @@ SafeFuture<Optional<SyncCommitteeDuties>> getSyncCommitteeDuties(
   SafeFuture<Optional<ProposerDuties>> getProposerDuties(final UInt64 epoch);
 
   SafeFuture<Optional<BeaconBlock>> createUnsignedBlock(
-      UInt64 slot, BLSSignature randaoReveal, Optional<Bytes32> graffiti, boolean blinded);
+      final UInt64 slot,
```

Comment:
nit: We omit the `final` on method args in interfaces (there's no implementation that could possibly reassign it so it's entirely redundant).

Reply:
kk, noted, I will remove all of them then since there were some on there before